src/test/java/com/example/raisetechcoursetask10/service/SkiresortServiceImplTest.java

```diff
@@ -113,4 +116,24 @@ class SkiresortServiceImplTest {
         verify(skiresortMapper,times(1)).findById(1);
         verify(skiresortMapper, times(1)).deleteSkiresort(1);
     }
+
+    @Test
+    public void 新規のスキー場情報を登録できること() {
+        // skiresortCreateForm変数をインスタンス化して、それぞれの属性に値を設定
+        SkiresortCreateForm skiresortCreateForm = new SkiresortCreateForm(1, "CoronetPeak", "NZ", "初中級者の時に行ったので初めてのTバーに撃沈。岩だらけの広い氷山には木が１本もなくて、ボードと心が折れる人続出。上手くなってから行くべきゲレンデ");
+        Skiresort skiresort = new Skiresort(0, "CoronetPeak", "NZ", "初中級者の時に行ったので初めてのTバーに撃沈。岩だらけの広い氷山には木が１本もなくて、ボードと心が折れる人続出。上手くなってから行くべきゲレンデ");
+
+        // スタブ化　insertSkiresortの戻り値はvoidのためdoNothingを使用する
```

Comment:
地味ですが全角スペースが混じっているのが気になります。
```suggestion
        // スタブ化 insertSkiresortの戻り値はvoidのためdoNothingを使用する
```

IntelliJには半角スペースや全角スペースなどのWhitespaceを可視化する方法があるので導入しておくと良いです。
VSCodeやターミナルにも同様の仕組みをいれておくと全角混じっててエラーだったというようなケアレスミスを防げます。

Reply:
参考
https://blog.marcnuri.com/intellij-idea-how-to-show-white-space

Reply:
ありがとうございます。
気づいていませんでしたが、私も気になります。
早速設定し、他にも最終行に自動改行を設定してくれる方法を見つけたので、色々設定する良い機会になりました。